core/object/script_language.h

```diff
@@ -68,7 +68,7 @@ class ScriptServer {
 	_FORCE_INLINE_ static int get_language_count() { return _language_count; }
 	static ScriptLanguage *get_language(int p_idx);
 	static void register_language(ScriptLanguage *p_language);
-	static void unregister_language(ScriptLanguage *p_language);
+	static void unregister_language(const ScriptLanguage *p_language);
```

Comment:
I assume it can't be done for the `register_language` function too?

Reply:
No it can't.